kraken-build/src/kraken/std/python/buildsystem/__init__.py

```diff
@@ -174,22 +174,31 @@ def detect_build_system(project_directory: Path) -> PythonBuildSystem | None:
         return PoetryPythonBuildSystem(project_directory)
 
     if "maturin" in pyproject_content:
-        if "[tool.poetry]" in pyproject_content:
+        if "[tool.poetry" in pyproject_content:
             from .maturin import MaturinPoetryPythonBuildSystem
 
             return MaturinPoetryPythonBuildSystem(project_directory)
-        else:
+        elif "[tool.pdm" in pyproject_content:
             from .maturin import MaturinPdmPythonBuildSystem
 
             return MaturinPdmPythonBuildSystem(project_directory)
+        else:
+            from .maturin import MaturinUvPythonBuildSystem
+
+            if "[tool.uv]" not in pyproject_content:
```

Comment:
```suggestion
            if "[tool.uv" not in pyproject_content:
```

Just like the checks for poetry/pdm?